### DISCUSSION TITLE
Some ideas 💡

### DISCUSSION THREAD

**Discussion by Tunsinasro** [spacecowboy/Feeder#189] [Ideas]
These are some features that are implemented in other RSS reader, it would be nice if they are also implemented in feeder
1. Double clicking on the article loads the complete article text (no need to go to the top menu)
2. Being able to zoom and/or download images inside the article (or at least opening them in separate window)
3. Showing a shortcut to the most used share option (ex pocket or instapaper) at the top after the title (or adding an option in the settings to make the shortcut permanent)

What do you think??

**Comment by spacecowboy** [comment]
1. Double clicking is not really a thing on touch screens
2. That would be nice
3. Feeder has no idea what you share to, so it can't show you this. And even if it could, I prefer to use the system's standard share sheet. Google at some point claimed that your most used services would be shown first here, but personally I've never seen that..

**Reply by Tunsinasro** [reply]
@spacecowboy 
1. Think web browsers, when you double click on the page it zooms in or out
3. Maybe not the most used share option, but integrating a read it later service like instapaper or pocket or even copying the url of the article to the clipboard

Thank you

**Reply by spacecowboy** [reply]
copying url is available in the standard share already
